.github/workflows/e2e.yml

```diff
@@ -0,0 +1,134 @@
+name: SalesforceCommerceCloud/pwa-kit/e2e
+on:
+  workflow_dispatch:
+  push:
```

Comment:
WIll remove the `push` and `workflow_dispatch` triggers later. I have no other way to run the workflow during development